socorro/unittest/processor/rules/test_mozilla.py

```diff
@@ -597,7 +597,7 @@ def test_everything_we_hoped_for(self):
         rule.act(raw_crash, raw_dumps, processed_crash, processor_meta)
 
         # The entire JavaStackTrace blob
-        assert processed_crash["java_stack_trace_full"] == raw_crash["JavaStackTrace"]
+        assert processed_crash["java_stack_trace_raw"] == raw_crash["JavaStackTrace"]
```

Comment:
I changed the tests to only test the new one. That makes for slightly less work in January 2020 when cleaning up.